src/api/db/data/20210707102506_add_api_endpoints_to_payload.rb

```diff
@@ -0,0 +1,21 @@
+class AddApiEndpointsToPayload < ActiveRecord::Migration[6.1]
+  def up
+    EventSubscription.where(channel: 'scm').each do |event_sub|
+      next if event_sub.payload['api_endpoint'].present?
+
+      if event_sub.payload['scm'] == 'github'
+        event_sub.payload['api_endpoint'] = 'https://api.github.com'
```

Comment:
Will this be correct if this migration is run *after* #11325?

Reply:
Yes, because we only update this if `event_sub.payload['api_endpoint']` is not available. After #11325 it always is available.